integration/tests/basic/test_deposit.py

```diff
@@ -190,3 +191,37 @@ def test_failed_withdraw_non_divisible_amount(
         destination_balance_after = spl_neon_token.get_balance(dest_acc.pubkey(), commitment=Commitment("confirmed"))
         with pytest.raises(AttributeError):
             _ = destination_balance_after.value
+
+    @pytest.mark.skip(reason="https://neonlabs.atlassian.net/browse/NDEV-3674")
+    def test_withdraw_wrapped(
+        self,
+        web3_client_sol: Web3Client,
+        account_with_all_tokens: LocalAccount,
+        solana_account: Keypair,
+        neon_mint: Pubkey,
+        sol_client: SolanaClient,
+    ):
+        contract, _ = web3_client_sol.deploy_and_get_contract(
+            contract="precompiled/NeonToken",
+            version="0.8.10",
+            account=account_with_all_tokens,
+        )
+
+        trx = Transaction()
+        trx.add(create_associated_token_account(solana_account.pubkey(), solana_account.pubkey(), neon_mint))
+        opts = TxOpts(skip_preflight=True, skip_confirmation=False)
+        sol_client.send_transaction(trx, solana_account, opts=opts)
+
+        associated_token_address = get_associated_token_address(solana_account.pubkey(), neon_mint)
+        balance_before = sol_client.get_balance(associated_token_address, commitment=Commitment.Confirmed).value
+
+        amount = 100
+        tx = web3_client_sol.make_raw_tx(from_=account_with_all_tokens, amount=amount)
+        instruction_tx = contract.functions.withdraw(bytes(associated_token_address)).build_transaction(tx)
+        receipt = web3_client_sol.send_transaction(account_with_all_tokens, instruction_tx)
+        assert receipt["status"] == 1
+
+        wait_condition(
+            lambda: sol_client.get_balance(associated_token_address, commitment=Commitment.Confirmed).value
```

Comment:
by this way you can get only SOL balance, not wSOL balance 

it will not change by withdraw